### DISCUSSION TITLE
why is the .isar db file larger than 100mb but there are only less than 100 datas.

### DISCUSSION THREAD

**Discussion by zhangwb1996** [isar/isar#1151] [Help]
there were 500K test datas (it was due to not run build__runner in terminal after changing the collection )  in isar,
1. clean isar from the `Isar Inspector`,
2. restart my App again,
3. inset less than 100 datas from UI of this App.

then, i find the  .isar db file is larger than 100mb only with less than 100 datas.
<img width="476" alt="image" src="https://user-images.githubusercontent.com/41967295/227417905-5fdc7716-b90f-493f-a7b3-0a8128265d3c.png">
<img width="100" alt="image" src="https://user-images.githubusercontent.com/41967295/227417956-fc4b7acd-024a-492d-ac3e-f9082a1e8544.png">

**Comment by Jtplouffe** [comment]
Hi, when deleting data from Isar, the underlying storage may not actually delete the bytes from the file immediately. This is done to improve performance.
Please refer to this issue for more info: https://github.com/isar/isar/issues/1043#issuecomment-1383146114.
